### PR TITLE
Add Magento projects to project type mapping

### PR DESCRIPTION
**Pull request by ArjenMiedema**
To make it easier to install the testing suite for Magento, it's
now possible to set the project type in the project's composer.json
file to `magento2-project` or `magento-project` to get the correct
coding standards files in the project.

### CODE REVIEW

**Review by leonhelmus — APPROVED** [review]

**Comment by leonhelmus** [comment]
Looks good i have also tested it. Just waiting for an approval of @marcelmediact .

File: src/ProjectTypeResolver.php [hunk 1]
```diff
@@ -53,6 +55,8 @@ public function __construct(Composer $composer, array $mapping = null)
     public function resolve(): string
     {
         $config = $this->composer->getConfig();
+
+        // Fallback to the "old" solution
```
Comment (ksangers) [inline src/ProjectTypeResolver.php:59]: This is not the fallback. This is the way to allow it to be configured (and overridden). Basing the testing-suite on the "type" is the fallback, even though it is very useful. Please remove or reflect this in the comment.

**Review by ksangers — CHANGES_REQUESTED** [review]

**Comment by ksangers** [comment]
@ArjenMiedema I'm afraid a small change is required.

**Comment by ksangers** [comment]
@ArjenMiedema I'm afraid I have a small request for change. To prevent it from being removed in the future.

File: src/ProjectTypeResolver.php (hunk 1 above)
Comment (ArjenMiedema) [reply src/ProjectTypeResolver.php:59]: True, that was an incorrect comment here. Removed it.

**Review by ksangers — APPROVED** [review]

**Pull request merged**